Sources/IO/Geometry/PLYReader/index.js

```diff
@@ -157,6 +163,27 @@ function postProcess(buffer, elements) {
     }
   }
 
+  if (hasFaceTCoods) {
```

Comment:
What if you have both `hasVertTCoords` and `hasFaceTCoods`?

Reply:
I have never seen such type of PLY.
Do you have any suggestion, use VertTCoords or use FaceTCoords?

Reply:
I don't think we can have such case

Reply:
Anyway, I prefer to take the VertTCoords in this case.